config/routes.rb

```diff
@@ -51,6 +51,11 @@
     resources :qc_files, controller: :qc_files
   end
 
+  scope 'tube_rack', controller: :tube_rack_creation do
+    get  'new', action: :tubes_to_rack, as: :new_tube_rack
+    post 'create', action: :create, as: :create_tube_rack
+  end
```

Comment:
This isn't quite right, especially as we are routing the show etc to limber_tube_racks. However I can't quite work out what the best pattern is, as it doesn't match up with the way thins work for plates etc. However, this is probably best deferred until when we actually have a use.

We could consider removing them for now? But easy either way.